docs/configuration.md

```diff
@@ -109,3 +108,18 @@ httpProxy = 'http://username:password@proxyAddress:port'
 ### Log file
 
 On XOA, the log file for XO-server is in `/var/log/syslog`. It contains all the server information returned and can be a real help when you have trouble.
+
+
+### SUDO
+
+To be able to mount NFS shares with a non-privileged user, the use of ``sudo`` is required. By default, XO-Server does not use sudo. To enable it, change the parameter ``useSudo`` to ``true``. It's located inside the `remoteOptions` block.
+
+```toml
+useSudo = true
+```
+
+You also need to configure ``sudo`` on your operating system to accept the user of your choice to run mount/umount commands without asking for a password. Depending on your operating system / sudo version, the location of this configuration may change. Regardless, you can use:
+
+```
+username ALL=(root)NOPASSWD: /bin/mount, /bin/umount
+```
```

Comment:
This is duplicated in *from the sources*.

As it only applies when XO is installed from the sources, I think we can remove it from here, what do you think?

Reply:
You're right, it is generic :slightly_smiling_face: 

It's still duplicated though, which section do you think is the more appropriate for this then?

Reply:
Yeah, I was thinking about that... Considering this will be used only by users installing from sources, may it should be there. But I think it's maybe weird the main configuration doc referrer to a specific explanation from a doc previously shown. I really don't know! Your call! :D

Reply:
IMHO running as `root` or not is an installation decision, so it would makes sense to put into *from the sources* section.

Reply:
So, may configuration.md should only warn users about running with non-privileged users and referrer to from_the_sources.md?

Reply:
I don't think so, it would make more sense to have this warning in *Backup troubleshooting*.

Reply:
I'm fine with this.